src/components/schedule-page/import-buttons/import-buttons.component.tsx

```diff
@@ -71,21 +77,18 @@ export function ImportButtonsComponent(): JSX.Element {
             setOpen(false);
           }}
         >
-          <ButtonGroup orientation="vertical">
-            <Button onClick={(): void => fileUpload.current?.click()}>
-              Wczytaj
-              <input
-                ref={fileUpload}
-                id="file-input"
-                data-cy="file-input"
-                onChange={(event): void => handleImport(event)}
-                style={{ display: "none" }}
-                type="file"
-                accept=".xlsx"
-              />
-            </Button>
-            <Button onClick={(): void => handleExport()}>Zapisz jako...</Button>
-          </ButtonGroup>
+          <div>
+            {DropdownButtonsComponent(btnData)}
+            <input
+              ref={fileUpload}
+              id="file-input"
+              data-cy="file-input"
+              onChange={(event): void => handleImport(event)}
+              style={{ display: "none" }}
+              type="file"
+              accept=".xlsx"
+            />
+          </div>
         </ClickAwayListener>
```

Comment:
Myślę, że ten `ClickAwayListener` też mógłby przejść do tego komponentu co ty zrobilaś, bo on odpowiada za to, żeby dało się zamknąć dropdown przy kliknięciu gdzieś poza nim

Reply:
Wrzuciłam do tego komponentu ClickAwayListener i nawet trochę więcej: myślę, że dzięki temu będzie można go bardziej ogólnie wykorzystywać i import-buttons.component też się trochę uprościł 😄

Reply:
Teraz jest zdecydowanie lepiej, super to zrobiłaś ;)